src/views/items/BootAppItem.ts

```diff
@@ -34,6 +34,10 @@ export class BootAppItem implements vscode.TreeItem {
         if (this._app.contextPath) {
             list.push(this._app.contextPath);
         }
+        if(this._app.activeProfiles) {
+            const profileInfo = `profile(s): '${this._app.activeProfiles}'`
```

Comment:
I would show the message as `profiles: ...`

Reply:
I thought about that, but for single profile, it would be in plural. But visually it looks good to have it as profiles as you stated. Changes pushed and that resolves all I hope.